doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,6 +1,7 @@
 # Upgrade Notes
 
 ## 6.9.0
+- [Documents] Announcement: Run the `pimcore:documents:migrate-elements` command before (!) you upgrade to Pimcore 10. Not needed if you don't intend to keep your document versions.   
```

Comment:
This PR targets `master` but adds an upgrade note to `v6.9.0` (which is already released and should probably be `v6.9.2` anyways).

Reply:
merci! The intention is that it is for 6.9